arangod/VocBase/Methods/Collections.cpp

```diff
@@ -263,14 +264,38 @@ Result Collections::create(TRI_vocbase_t& vocbase,
                info.collectionType != TRI_col_type_e::TRI_COL_TYPE_EDGE) {
       return TRI_ERROR_ARANGO_COLLECTION_TYPE_INVALID;
     }
+
     TRI_ASSERT(info.properties.isObject());
     helper.clear();
     helper.openObject();
     helper.add(arangodb::StaticStrings::DataSourceType,
                arangodb::velocypack::Value(static_cast<int>(info.collectionType)));
     helper.add(arangodb::StaticStrings::DataSourceName,
                arangodb::velocypack::Value(info.name));
+
+
+    auto replicationFactorSlice = info.properties.get(StaticStrings::ReplicationFactor);
+    if(replicationFactorSlice.isNone()) {
+      auto factor = vocbase.replicationFactor();
+      if(factor > 0 && vocbase.IsSystemName(info.name)) {
+        auto* cl = application_features::ApplicationServer::lookupFeature<ClusterFeature>("Cluster");
```

Comment:
use `getFeature` to prevent nullptr access
```suggestion
        auto* cl = application_features::ApplicationServer::getFeature<ClusterFeature>("Cluster");
```

Reply:
The cluster feature is not available in all unit tests. Making features mandatory increases the complexity of testing. Do we really want that?